test/Winton.DomainModelling.DocumentDb.IntegrationTests/ValueObjectFacadeFactoryTests.cs

```diff
@@ -13,30 +13,31 @@
 
 namespace Winton.DomainModelling.DocumentDb
 {
-    public class ValueObjectFacadeTests : IDisposable
+    public class ValueObjectFacadeFactoryTests : IDisposable
```

Comment:
I think this should still be named `ValueObjectFacadeTests` as I think that's actually what is being integration tested here, even though you obviously have to access it through the factory because the implementation is internal.